### PR TITLE
Formatting scripts array inputs

### PR DESCRIPTION
**Pull request by avidan-H**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/15650
fixes: https://github.com/demisto/etc/issues/16203
fixes: https://github.com/demisto/etc/issues/16190
fixes: https://github.com/demisto/etc/issues/16426

## Description
Alter the following scripts to accept array input
ExtractDomainFromUrlAndEmail
UnEscapeIPs
UnEscapeURLs

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [ ] Documentation (with link to it)
- [x] Code Review

## Dependencies
- N/A

## Additional changes
Added test playbooks for UnEscapeIPs, ExtractDomainFromUrlAndEmail, and updated the test playbook for UnEscapeURLs

### CODE REVIEW

File: Scripts/script-ExtractDomainFromUrlFormat.yml [hunk 1]
```diff
@@ -7,25 +7,34 @@ script: |-
...
-             demisto.results("")
-      else:
-          demisto.results(domain)
+  if isinstance(the_input, str) or isinstance(the_input, unicode):
+      the_input = argToList(the_input)
+
+  # Otherwise assumes it's an array
+  for item in the_input:
+      domains.append(extract_domain(item))
+  demisto.results(domains)
```
Comment (glicht) [inline Scripts/script-ExtractDomainFromUrlFormat.yml]: using results with the array is not good as it creates an entry for each element in the array. we want to return one array with all results. You should change to return a single entry of the form:
```
demisto.results({
      'Type': entryTypes['note'],
      'ContentsFormat': formats['json'],
      'Contents': domains,      
  })
```

Also note: when a single param is passed in (not array) we should return a single string in the Content value and not an array with a single string.

**Comment by glicht** [comment]
See comments at original issue. Also, need to merge in recent changes done to ExtractDomainFromUrlFormat

**Comment by avidan-H** [comment]
UnEscapeURL-Test playbook
image

UnEscapeIPs-Test playbook
image

ExtractDomainFromUrlAndEmail-Test playbook
image

**Comment by anara123** [comment]
@Itay4 @glicht what is the status here? can we approve?

**Pull request merged**